languages/move_on_aptos/generic/Parse_move_on_aptos_tree_sitter.ml

```diff
@@ -1105,18 +1137,18 @@ let map_specifier (env : env) (x : CST.specifier) : G.attribute list =
                    G.Arg (G.L (G.Bool (false, tok)) |> G.e)
                | None -> G.Arg (G.L (G.Bool (true, sc)) |> G.e)
              in
-             let specifier =
+             let tk, specifier =
                match v2 with
-               | `Acquis tok -> (* "acquires" *) str env tok
-               | `Reads tok -> (* "reads" *) str env tok
-               | `Writes tok -> (* "writes" *) str env tok
+               | `Acquis tok -> (token env tok, (* "acquires" *) str env tok)
+               | `Reads tok -> (token env tok, (* "reads" *) str env tok)
+               | `Writes tok -> (token env tok, (* "writes" *) str env tok)
```

Comment:
`str` just returns `string * Tok.t`, where the second item is just the result of a call to `token`. You can simplify this by removing the call to `token` and just calling `snd specifier` where needed (or destructuring the tuple if you prefer)